src/button.tsx

```diff
@@ -504,6 +483,8 @@ interface ButtonLinkHrefProps extends ButtonLinkCommonProps {
 
 interface ButtonLinkToProps extends ButtonLinkCommonProps {
     to: string;
+    newTab?: boolean;
+    /** @deprecated use newTab */
     fullPageOnWebView?: boolean;
```

Comment:
are you sure? how can you replicate `fullPageOnWebview` behavior with `newTab`?

Reply:
newTab={inApp()}